hu.elte.refjava.lang/src/hu/elte/refjava/api/LocalRefactoring.xtend

```diff
@@ -58,14 +68,18 @@ class LocalRefactoring implements Refactoring {
 			false
 		}
 	}
+	
+	def protected void setMetaVariables() {
+		//empty
+	}
```

Comment:
It might be cleaner to make methods like these abstract instead of defining them with an empty body.

Reply:
It couldn't be done.